apps/mobile/src/app/components/AuctionCreate/AuctionCreatePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { AuctionCreateTemplate } from './AuctionCreateTemplate';
+import { useForm } from 'react-hook-form';
+import { AuctionFormValues } from './auction-create-form-values';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { apiRoutes } from '@inheartive/data';
+import { useNavigate } from 'react-router-native';
+import { RoutingPath } from '../../routing';
+
+export function AuctionCreatePage() {
+  const navigate = useNavigate();
+  const {
+    isLoading: categoriesIsLoading,
+    isError: categoriesIsError,
+    data: categories,
+  } = useQuery({
+    queryKey: ['categories'],
+    queryFn: () => fetch(apiRoutes.categories).then((res) => res.json()),
+  });
+
+  // tmp users loading since we don't have auth user right now
+  const { data: users } = useQuery({
+    queryKey: ['users'],
+    queryFn: () => fetch(apiRoutes.users).then((res) => res.json()),
+  });
+
+  const mutation = useMutation({
+    mutationFn: (data) => {
+      return fetch(apiRoutes.auctions, {
```

Comment:
We could consider using Axios or some other library since "fetch" makes code a little bit harder to read. Anyway this could be simplified by extracting to function,like "makeJsonPost" with args data and url.

Reply:
Should I do it now?

Reply:
I created a task for it :)